src/client.cpp

```diff
@@ -394,6 +414,22 @@ ADDON_STATUS ADDON_SetSetting(const char *settingName, const void *settingValue)
     XBMC->Log(LOG_INFO, "Changed setting 'enableoldseriesdlg' from %u to %u", g_bEnableOldSeriesDlg, *(bool*)settingValue);
     g_bEnableOldSeriesDlg = *(bool*)settingValue;
   }
+  else if (str == "keepmethodtype")
+  {
+	  if (g_KeepMethodType != *(TvDatabase::KeepMethodType*)settingValue)
+	  {
+		  XBMC->Log(LOG_INFO, "Changed setting 'keepmethodtype' from %u to %u", g_KeepMethodType, *(int*)settingValue);
+		  g_KeepMethodType = *(TvDatabase::KeepMethodType*)settingValue;
+	  }
+  }
+  else if (str == "defaultlifetime")
```

Comment:
Maybe it is bettter to change "defaultlifetime" into "defaultrecordinglifetime", because it is a recording setting.